tests/utils.py

```diff
@@ -2,22 +2,28 @@
 
 
 def assert_ops_equal(a, b, compare_fn=lambda a, b: a.op.type == b.op.type):
+    # TODO: Add a test that determines if a is a meta object
     if hasattr(a, 'op') or hasattr(b, 'op'):
         assert hasattr(a, 'op') and hasattr(b, 'op')
 
         assert compare_fn(a, b)
 
-        assert len(a.op.inputs) == len(b.op.inputs)
+        # grabing updated input representation
+        new_inputs = b.op._reconstruct_sequence_inputs(b.op.op_def,
```

Comment:
As the TODO above states, you need to check whether or not the inputs even need to be restructured (i.e. check if `a` and/or `b` is a TF tensor and *then* restructure accordingly).